admin/taxonomy/class-taxonomy.php

```diff
@@ -379,6 +379,8 @@ private function get_replace_vars() {
 			'tag_description',
 			'searchphrase',
 			'currentyear',
+			'currentmonth',
+			'currentday',
```

Comment:
Maybe we should add `'permalink'` as a replacement var for terms?

Doing so would require enhancing the `retrieve_permalink()` function

Reply:
We're focusing on posts only right now so I undid this change.